### PR TITLE
Piexif

### PR DESCRIPTION
**Pull request by agcopenhaver**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/piexif/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+  host:
+    - python
+    - pip
+    - Pillow==5.4.1
```
Comment (mariusvniekerk) [inline recipes/piexif/meta.yaml]: remove these unused host deps please

File: recipes/piexif/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  sha256: 1fd0e5d21e9a69be99205979b927b1803c2daae06a4e54a802b0d0d520932722
+
+build:
+
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
```
Comment (mariusvniekerk) [inline recipes/piexif/meta.yaml]:
```suggestion
```

File: recipes/piexif/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+  host:
+    - python
+    - pip
+    - Pillow==5.4.1
+    - Sphinx
+    - sphinxcontrib-websupport
+  run:
+    - python
+    - Pillow==5.4.1
```
Comment (mariusvniekerk) [inline recipes/piexif/meta.yaml]: You may want this pin to be less strict
```suggestion
    - pillow =5.4.1
```

File: recipes/piexif/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    - Sphinx
+    - sphinxcontrib-websupport
+test:
+  imports:
+    - copy
```
Comment (mariusvniekerk) [inline recipes/piexif/meta.yaml]: Remove the import tests for standard library pieces and pil please

**Comment by agcopenhaver** [comment]
Ok - I think that should do it

File: recipes/piexif/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/hMatoba/Piexif/archive/1.1.2.tar.gz
+  sha256: 1fd0e5d21e9a69be99205979b927b1803c2daae06a4e54a802b0d0d520932722
+
+build:
+
```
Comment (marcelotrevisani) [inline recipes/piexif/meta.yaml]: please remove this empty new line

File: recipes/piexif/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,50 @@
+{% set name = "Piexif" %}
+{% set version = "1.1.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/hMatoba/Piexif/archive/1.1.2.tar.gz
```
Comment (marcelotrevisani) [inline recipes/piexif/meta.yaml]: Please, change it to
```
  url: https://github.com/hMatoba/Piexif/archive/{{ version }}.tar.gz
```

**Comment by agcopenhaver** [comment]
Ok - I've made those changes too.

File: recipes/piexif/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - pillow=5.4.1
```
Comment (marcelotrevisani) [inline recipes/piexif/meta.yaml]: please change it to
```
pillow 5.4.1
````

**Comment by agcopenhaver** [comment]
Done

**Pull request merged**